src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar
+
+class Car(val order: Int) {
+    private val movements = mutableListOf<Int>()
+
+    fun getCurrentPosition(): Int {
+        return movements.sum()
+    }
+
+    /**
+     * get position at  n th trial
+     * trialNumber : n th trial (starting from 1)
+     */
+    fun getPositionAt(atTrial: Int): Int {
+        require(atTrial > 0 && atTrial <= movements.size) { "Invalid trial number $atTrial" }
```

Comment:
```suggestion
        require(atTrial in 1..movements.size) { "Invalid trial number $atTrial" }
```
코틀린에서는 이렇게도 표현할 수 있습니다 😄